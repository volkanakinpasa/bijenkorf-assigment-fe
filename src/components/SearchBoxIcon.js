function SearchBoxIcon() {
  return <img src="search-icon.svg" />;
}

export default SearchBoxIcon;
