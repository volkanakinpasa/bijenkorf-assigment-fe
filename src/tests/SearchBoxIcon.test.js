import SearchBoxIcon from '../components/SearchBoxIcon';
import { render } from '@testing-library/react';

describe('SearchBoxIcon tests', () => {
  it('creates a snapshot', () => {
    const { asFragment } = render(<SearchBoxIcon />);
    expect(asFragment()).toMatchSnapshot();
  });
});
