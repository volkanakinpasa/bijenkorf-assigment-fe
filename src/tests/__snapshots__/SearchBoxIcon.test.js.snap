// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBoxIcon tests creates a snapshot 1`] = `
<DocumentFragment>
  <img
    src="search-icon.svg"
  />
</DocumentFragment>
`;
